Applications/DataExplorer/DataView/DiagramView/CMakeLists.txt

```diff
@@ -39,6 +39,12 @@ add_library(QtDiagramView
     ${UIS}
 )
 
-target_link_libraries(QtDiagramView Qt4::QtGui)
+target_link_libraries(QtDiagramView
+    Qt4::QtGui
+    logog
+    BaseLib
+    GeoLib
```

Comment:
`logog` is linked in `BaseLib` which is linked in `MathLib` which is linked in `GeoLib`. Maybe it is enough to link against `GeoLib`?